sbc_translation/models/correspondence.py

```diff
@@ -22,22 +33,119 @@ class Correspondence(models.Model):
     ##########################################################################
     #                              ORM METHODS                               #
     ##########################################################################
+
     @api.model
     def create(self, vals):
-        """ Create a message for sending the CommKit only if the preferred
-            language is the same as the letter's language else send the letter
-            on the local translate plaforme.
-            """
-
+        """ Create a message for sending the CommKit after be translated on
+             the local translate plaforme.
+        """
         sponsorship = self.env['recurring.contract'].browse(
             vals['sponsorship_id'])
-        if sponsorship.reading_language.id == vals['original_language_id']:
-            letter = super(Correspondence, self).create(vals)
-        else:
+
+        original_lang_id = self.env['res.lang.compassion'].browse(
+            vals['original_language_id'])
+
+        if original_lang_id.translatable:
```

Comment:
I think this test is wrong. If we can translate a language does not mean that we have to! For instance if the letter is in french, `translatable` will be `True`, but if the letter is for Togo, we don't need to translate it.